src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -70,6 +70,18 @@ def test_list_success(resource_client, current_resource_model):
     return resource_models
 
 
+def test_list_success(resource_client, current_resource_model):
+    resource_models = get_resource_model_list(resource_client, current_resource_model)
+    result = False
+    for resource_model in resource_models:
```

Comment:
can use all() here like you did for assert functions

Reply:
I am re using this method for both success and failure. Therefore, I don't want to assert. I just wanted to return the value. This just decreased replicating code.

Reply:
not assert, just suggesting to use 

```
all(resource_client.does_primary_identifier_exist(
            resource_client.primary_identifier_paths,
            resource_model,
            current_resource_model,
        ) for model in resource models)
```

to keep codebase consistent. should be the same thing if youre trying to test if every model has primary identifiers. 

Why are we using bitwise operators here?

Reply:
> Why are we using bitwise operators here?

+1, was also confused by these, would prefer `and`/`or`

Reply:
ok. I think I misunderstood. Will add all